porcupine/settings.py

```diff
@@ -223,22 +227,24 @@ def set(
 
         You can specify ``call_converter`` to force the converter to be or
         to not be called.
-        """
-        # ...even though this method isn't named 'set_'. But the docstring is
-        # used in settings.rst to document a global "function".
 
+        If ``tag`` is given, the option name will be returned from
+        :meth:`get_options_by_tag`. Otherwise the tag of the option is cleared.
```

Comment:
```suggestion
        Each option has a tag that shows up in :meth:`debug_dump` and :meth:`get_options_by_tag`.
        If the ``tag`` argument is given, the tag of the option will be changed to it.
        Otherwise the tag is cleared.
        Tagging is useful for figuring out where/why an option was set:
        if you call ``.set()`` in two places with different tags,
        the option's current tag tells which of the two places has set the current value.
```